### PR TITLE
Add doc updates for predisposing/oncogenic eids/assertions.

### PR DESCRIPTION
**Pull request by acoffman**
(Arpad did all the hard work here, I just made the PR)

### CODE REVIEW

File: docs/curating/evidence.rst [hunk 1]
```diff
@@ -117,58 +117,83 @@ Prognostic evidence is characterized by either better outcomes for patient subpo
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Oncogenic Evidence Type describes literature-derived evidence pertaining either to a somatic variant’s protective effects or its role in tumor formation, growth, survival or metastasis, as summarized by Hanahan and Weinberg in `Hallmarks of Cancer <https://pubmed.ncbi.nlm.nih.gov/21376230/>`__.. Disease type should be specified, as oncogenic effects may depend on cellular context. For cases where a disease type is difficult to ascertain, such as experiments in highly de-differentiated cell lines, the Disease Ontology term ‘Cancer’ can be used. The Evidence Statement should contain a summary of the experiments or findings suggesting a protective, oncogenic, or benign variant effect.
```
Comment (tmooney) [inline docs/curating/evidence.rst:182]:
```suggestion
The Oncogenic Evidence Type describes literature-derived evidence pertaining either to a somatic variant’s protective effects or its role in tumor formation, growth, survival or metastasis, as summarized by Hanahan and Weinberg in `Hallmarks of Cancer <https://pubmed.ncbi.nlm.nih.gov/21376230/>`__. Disease type should be specified, as oncogenic effects may depend on cellular context. For cases where a disease type is difficult to ascertain, such as experiments in highly de-differentiated cell lines, the Disease Ontology term ‘Cancer’ can be used. The Evidence Statement should contain a summary of the experiments or findings suggesting a protective, oncogenic, or benign variant effect.
```

File: docs/curating/evidence.rst [hunk 2]
```diff
@@ -117,58 +117,83 @@ Prognostic evidence is characterized by either better outcomes for patient subpo
...
+The Oncogenic Evidence Item may be associated with `Oncogenicity Codes <https://pubmed.ncbi.nlm.nih.gov/35101336/>`__. developed by the Knowledge Curation and Interpretation Standards (KCIS) working group of the GA4GH VICC in collaboration with ClinGen working groups. Oncogenicity codes assess oncogenicity of a given somatic variant in a mechanism similar to that used in the 2015 ACMG/AMP Guidelines for germline pathogenicity. Enumeration of Oncogenicity Codes derived from the literature along with a brief justification for the presence of each code can be included in the Evidence Statement.
```
Comment (tmooney) [inline docs/curating/evidence.rst:184]:
```suggestion
The Oncogenic Evidence Item may be associated with `Oncogenicity Codes <https://pubmed.ncbi.nlm.nih.gov/35101336/>`__ developed by the Knowledge Curation and Interpretation Standards (KCIS) working group of the GA4GH VICC in collaboration with ClinGen working groups. Oncogenicity codes assess oncogenicity of a given somatic variant in a mechanism similar to that used in the 2015 ACMG/AMP Guidelines for germline pathogenicity. Enumeration of Oncogenicity Codes derived from the literature along with a brief justification for the presence of each code can be included in the Evidence Statement. 
```

**Review by tmooney — COMMENTED** [review]
Most of these edits are removing `.`s from the end of hyperlinks. It happens enough that I wonder if I'm missing something and those `.`s are important somehow?

Besides that, a few commas and a `-` for good measure!

**Review by malachig — APPROVED** [review]
Looks good to me.

**Review by obigriffith — APPROVED** [review]
Looks good. Let's just get this live.

**Pull request merged**